src/Srclib/Converter.hs

```diff
@@ -63,9 +63,14 @@ mkSourceUnitDependency gr locator = SourceUnitDependency
   , sourceDepImports = Set.toList $ AM.postSet locator gr
   }
 
-isProdDep :: Dependency -> Bool
-isProdDep Dependency{dependencyEnvironments} =
-  null dependencyEnvironments || EnvProduction `elem` dependencyEnvironments
+shouldPublishDep :: Dependency -> Bool
+shouldPublishDep Dependency{dependencyEnvironments} =
+  null dependencyEnvironments || EnvProduction `elem` dependencyEnvironments || hasOtherEnv dependencyEnvironments
+
+hasOtherEnv :: [DepEnvironment] -> Bool
```

Comment:
[nit] let's narrow this to:

```hs
isOtherEnv :: DepEnvironment -> Bool
```

and use `any :: (a -> Bool) -> [a] -> Bool` in `shouldPublishDep`